src/helpers/augmentedReality/storageNameCreator.js

```diff
@@ -1,6 +1,13 @@
+import { documentDirectory } from 'expo-file-system';
+
 export const storageNameCreator = ({ dataItem, objectItem }) => {
-  const { id: objectId } = dataItem;
-  const { title, type, id } = objectItem;
+  const { id: objectId, title: objectTitle } = dataItem;
+  const { title, type } = objectItem;
+  const replacedObjectTitle = objectTitle.replace(/\s+/g, '');
```

Comment:
what does `replacedObjectTitle` tell us? could this be something more meaningful? like `titleWithoutSpaces`?

Reply:
Yeah, it's better now.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/569/commits/1dbcc5eb3ef7c049a142104e92b97fe6a1ba2bda

Reply:
Yeah, it's better now.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/569/commits/881f40de6376a8c53355e44d9c6fe9d8dc3749f8